src/cpu-template-helper/src/template/dump/aarch64.rs

```diff
@@ -20,38 +32,47 @@ pub fn config_to_template(cpu_config: &CpuConfiguration) -> CustomCpuTemplate {
 #[cfg(test)]
 mod tests {
     use vmm::arch::aarch64::regs::Aarch64Register;
+    use vmm::arch::regs::{Aarch64RegisterVec, RegData128};
 
     use super::*;
 
-    fn build_sample_regs() -> Vec<Aarch64Register> {
-        vec![
-            Aarch64Register {
-                id: 0x0,
-                value: 0xffff_ffff_ffff_ffff_ffff_ffff_ffff_ffff,
-            },
-            Aarch64Register {
-                id: 0xffff_ffff_ffff_ffff,
-                value: 0x0000_ffff_0000_ffff_0000_ffff_0000_ffff,
-            },
-            Aarch64Register {
-                id: 0x1,
-                value: 0x0000_ffff_0000_ffff_0000_ffff_0000_ffff,
-            },
-        ]
+    const KVM_REG_SIZE_U128: u64 = 0x0040000000000000;
```

Comment:
Is this the register ID where the u128 registers start? If so, we may need to name it a bit closer.

Reply:
Basically to create a `Aarch64Register` I need an ID which corresponds to the 128 bit wide reg. 
`KVM_REG_SIZE_U128` is technically not a valid register ID.

Reply:
But it is still used as an ID later, isn't it? Can we name it to something explaining that? Maybe
```rust
const KVM_REG_SIZE_U128: u64 = 0x0040000000000000;
const U128_REG_ID_BASE: u64 = KVM_REG_SIZE_U128;
```
or something?

Reply:
I added a comment for this.